src/pack-externals.ts

```diff
@@ -238,6 +238,12 @@ export async function packExternalModules(this: EsbuildPlugin) {
     }
   }
 
+  // GOOGLE: Copy modules only if not google-cloud-functions
```

Comment:
@floydspace what do you think of this ? I think google users won't need the (long) installation time, if it's only to remove `node_modules` right after ?

Reply:
hey @olup, I had experience with gcp neither, unfortunately (it's a heritage of `serverless-plugin-typescript`), but what you're saying is logical, seems we only need to prepare `package.json`, let's make an investigation issue for this, to keep in mind.